openedx_caliper_tracking/tests/local_test_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# This script file is to run the caliper app tests locally.
+
+
+FOLDER_NAME="caliper_app_testing"
+PROJECT_NAME="caliper_project"
+rm -rf $FOLDER_NAME
+mkdir $FOLDER_NAME
+cd $FOLDER_NAME
+
+printf "\n\n\n=========================================\n"
+printf "Making and Activating Virtual Environment"
+printf "\n=========================================\n"
+virtualenv venv --python=python2.7
+source venv/bin/activate
+
+printf "\n\n\n=======================\n"
+printf "Installing Dependencies"
+printf "\n=======================\n"
+pip install celery==4.3.0
+pip install Django==1.11.21
+pip install isodate==0.6.0
+pip install kafka-python==1.4.7
+pip install mock==3.0.5
+pip install python-dateutil==2.8.0
+pip install requests==2.22.0
+pip install coverage
+pip install factory-boy==2.12.0
+pip install git+https://github.com/ucsd-ets/openedx-caliper-tracking.git@umar/make-caliper-tests-independent
+
+printf "\n\n\n=======================\n"
+printf "Creating Django Project"
+printf "\n=======================\n"
+django-admin.py startproject $PROJECT_NAME
+sudo apt-get install sed
+cd $PROJECT_NAME/$PROJECT_NAME/
+# Append 'openedx_caliper_tracking' in INSTALLED_APPS list of settings.py
+sed "/'django.contrib.staticfiles',/a 'openedx_caliper_tracking'" settings.py > temp_settings.py
+rm settings.py
+mv temp_settings.py settings.py
+cd ..
+
+printf "\n\n\n=====================\n"
+printf "Running Caliper Tests"
+printf "\n=====================\n"
+coverage run --source='openedx_caliper_tracking' manage.py test openedx_caliper_tracking
+coverage report --omit='*tests*, *pavement.py*, *__init__.py*'
+deactivate
+cd ../..
+rm -rf $FOLDER_NAME
```

Comment:
I think we should make this code a function and make a `Makefile` with linting, quality checking and testing targets. Through that MakeFile we can trigger this testing script.